dapps/src/tests/helpers/fetch.rs

```diff
@@ -96,11 +98,8 @@ impl FakeFetch {
 impl Fetch for FakeFetch {
 	type Result = Box<Future<Item = fetch::Response, Error = fetch::Error> + Send>;
 
-	fn new() -> Result<Self, fetch::Error> where Self: Sized {
-		Ok(FakeFetch::default())
-	}
-
-	fn fetch_with_abort(&self, url: &str, _abort: fetch::Abort) -> Self::Result {
+	fn fetch(&self, url: &str, abort: fetch::Abort) -> Self::Result {
+		let u = Url::parse(url).unwrap();
```

Comment:
Perhaps `Url` crate should be re-exported from `fetch` if it's required to instantiate some other public types (`Response`)?

Reply:
Yes, this may be convenient.